Package.swift

```diff
@@ -115,7 +115,10 @@ let package = Package(
         ),
         .target(
             name: "CNIOLinux",
-            dependencies: []
+            dependencies: [],
+            cSettings: [
+                .define("_GNU_SOURCE"),
```

Comment:
Are we confident we're allowed to do this? Previously this has appeared to conflict with Swift's glibc module, which was declared without this definition.

Reply:
Hmmm. Well, we're definitely _not_ safe just `#define`ing it in a source file, because with modular includes you have to define it on the command line (or it won't affect the modularized headers). If it doesn't work with the Glibc module, that would be… bad.